contentcuration/contentcuration/frontend/shared/views/TipTapEditor/TipTapEditor/components/EditorToolbar.vue

```diff
@@ -170,11 +335,42 @@
         listFormatting$,
         scriptFormatting$,
         insertTools$,
+        clearFormatting$,
       } = getTipTapEditorStrings();
 
+      // Compute which categories should be visible vs in overflow
+      const visibleCategories = computed(() => {
+        const visible = [];
+        OVERFLOW_CATEGORIES.forEach(category => {
+          if (toolbarWidth.value >= OVERFLOW_BREAKPOINTS[category]) {
+            visible.push(category);
+          }
+        });
+        return visible;
+      });
+
+      const overflowCategories = computed(() => {
+        return OVERFLOW_CATEGORIES.filter(category => !visibleCategories.value.includes(category));
+      });
```

Comment:
I notice that you use `OVERFLOW_CATEGORIES.filter` in the `overflowCategories` property, but you use `forEach` and push to a local variable in the `visibleCategories`'s function. 

I suggest using filter in the `visibleCategories` function as well. Not a blocker - my suggestion here is motivated like 25% by how it'd be more consistent and 75% that it'd make the `visibleCategories` definition more concise and a bit more readable.